zstacklib/zstacklib/utils/iproute.py

```
@@ -73,38 +80,46 @@ def _get_scope_name(scope, exception_if_wrong = False):
+
 class NamespaceAlreadyExists(RuntimeError):
     def __init__(self, namespace):
-        super(NamespaceAlreadyExists, self).__init__("Network namespace : %(namespace)s has already exists." % {'namespace': namespace})
+        super(NamespaceAlreadyExists, self).__init__(
+            "Network namespace : %(namespace)s has already exists." % {'namespace': namespace})
         self.namespace = namespace
 
+
 class InvalidScope(RuntimeError):
     def __init__(self, scope):
         super(NoSuchScope, self).__init__("Scope : %(scope)s is invalid." % {'scope': scope})
```

Ruff (0.11.9)

Undefined name `NoSuchScope`

(F821)